src/main/java/com/atlassian/bitbucket/jenkins/internal/link/BitbucketExternalLinkUtils.java

```diff
@@ -0,0 +1,78 @@
+package com.atlassian.bitbucket.jenkins.internal.link;
+
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketPluginConfiguration;
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketServerConfiguration;
+import com.atlassian.bitbucket.jenkins.internal.scm.BitbucketSCMRepository;
+import hudson.util.FormValidation;
+import okhttp3.HttpUrl;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * @since 2.1.4
+ */
+@Singleton
+public class BitbucketExternalLinkUtils {
+
+    @Inject
+    private BitbucketPluginConfiguration bitbucketPluginConfiguration;
```

Comment:
totally up to you, but maybe move the `@Inject` to the constructor and remove the default constructor, that way you don't need two, and it should work (I imagine)